cmd/internal/client.go

```diff
@@ -51,3 +55,11 @@ func CreateMetadataClient(logger logr.Logger, opts ...meta.NewOption) metadata.I
 	checkError(logger, err, "failed to create metadata client")
 	return client
 }
+
+func CreateDClient(logger logr.Logger, ctx context.Context, dyn dynamic.Interface, kube kubernetes.Interface, resync time.Duration) dclient.Interface {
+	logger = logger.WithName("d-client")
+	logger.Info("create d client...", "kubeconfig", kubeconfig, "qps", clientRateLimitQPS, "burst", clientRateLimitBurst)
```

Comment:
Maybe good to change to "create dynamic client...".

Reply:
Updated.

Reply:
Yeah, but we already have a `dynamic client` thing, the one from kube ;)

Reply:
Ops, I did not check that. Any other suggestions?

Reply:
I was short on ideas here :(

Reply:
But I definitely agree the name sucks.

Reply:
"the Kyverno dynamic client"? At least if differs from the other one.